cmd/governor/app/nodereaper.go

```diff
@@ -63,4 +63,5 @@ func init() {
 	nodeReapCmd.Flags().StringVar(&nodeReaperArgs.ReapUnjoinedKey, "reap-unjoined-tag-key", "", "BE CAREFUL! EC2 tag key that identfies a joining node")
 	nodeReapCmd.Flags().StringVar(&nodeReaperArgs.ReapUnjoinedValue, "reap-unjoined-tag-value", "", "BE CAREFUL! EC2 tag value that identfies a joining node")
 	nodeReapCmd.Flags().StringArrayVar(&nodeReaperArgs.ReapTainted, "reap-tainted", []string{}, "marks nodes with a given taint reapable, must be in format of comma separated taints key=value:effect, key:effect or key")
+	nodeReapCmd.Flags().Float64Var(&nodeReaperArgs.ReconsiderUnreapableAfter, "reconsider-unreapable-after", 10, "Time (in minutes) after which reconsider unreapable nodes")
```

Comment:
Is 10 minutes a good default here? I assume people run node-reaper in different intervals (every 10,15 minutes etc) and if we keep hitting the same bad node that fails to drain it might mean we are not killing other nodes which are reapable.
Maybe 45-60 minutes is more appropriate for default backoff considering that prior to this PR it was indefinite?

Reply:
I will update it to be 45 minutes.